src/Panel/Panel.php

```diff
@@ -119,7 +126,7 @@ protected static function garbage(): void
 	 */
 	public static function go(string|null $url = null, int $code = 302): void
 	{
-		throw new Redirect(static::url($url), $code);
+		throw new Redirect(App::instance()->panel()->url($url), $code);
 	}
```

Comment:
Do you think it would break too much if we convert this to non-static as well?

Reply:
But maybe we are having this discussion too early on. We can still decide and change this later.